draft/webxdc-user-guide.md

```diff
@@ -0,0 +1,156 @@
+# Webxdc User Guide
+
+## Webxdc File Format
+
+- a **Webxdc app** is a **ZIP-file** with the extension `.xdc`
+- the ZIP-file must contain at least the file `index.html`
+- if the Webxdc app is started, `index.html` is opened in a restricted webview
+  that allow accessing resources only from the ZIP-file
+
+
+## Webxdc API
+
+Webxdc apps can do anything that is allowed in webviews -
+html, css, images, canvas, javascript and so on.
```

Comment:
but no web-requests and the actual webview functionality is dependend on it's version and the system version

Reply:
i just removed these lines, above is already stated that the webview is restricted to the ZIP - and a more coarse overview is available in webxdc.md - no need to repeat things,